public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+
+### Key Concepts
+
+Each `key concept` focuses on the position of the `party` along an important axis. The `disposition` of the party in these categories may be represented as the sum of positive `supporting details`, subtracting any negative `supporting details`, resulting in a range from -4 to +4. 
+
+`Supporting details` are specific people, places, things or ideas related to the party in some sort of way, while `key concepts` are overarching, category level descriptions of the party’s standing. 
+
+* **Assets** - This represents the available resources of the party, including money and any available equipment.
+* **Allies** - This represents allies to the party, which might be specific key contacts or whole organizations ready to back the party.
+* **Adversaries** - This represents the adversaries of the party, which could be specific powerful enemies or a whole civilization bent on your destruction.
+* **Authority** - This represents the party’s relationship with authority - how much they have, or how much those who have it like (or dislike) the party.
+
+Examples for each `key concept` and related `supporting details` are available further on in this document.
+
+### Dispositions 
+
+`Supporting details` have either a positive disposition (good for players, +1 to key concept), or a neutral disposition (+0 to key concept), or a negative disposition (bad for players, -1 to key concept). These dispositions can be used to alter the `position` of a roll. `Disposition` can alter the consequence rating of a `position`, and might be directly used by the GM to help set the `position` of a roll when it makes sense.
+
+
+| Disposition                  | Default Position                 | Consequence Rating               | 
+| ---------------------------- | -------------------------------- | -------------------------------- |
+| Positive (+)                 | Controlled                       | -1 Consequence Rating            |
+| Neutral  (0)                 | Risky                            | No change to Consequence Rating  |
+| Negative (-)                 | Desperate                        | +1 Consequence Rating            |
+
+The collection of `supporting details` for a `key concept` ladder up to a numeric score between -4 and +4. A `key concept` is considered neutral until the score reaches either <-2 (three or more negative `supporting details`), becoming a negative `disposition`, or >+2 (three or more positive `supporting details`), becoming a positive `disposition`. When the `key concept` changes disposition it should be rewritten to represent the change in fortunes for the party.
```

Comment:
question: why does it take -2 or +2 to become positive or negative? I feel like it adds complexity on how to calculate the rating of a key detail.

Reply:
I'm going to try to change the wording above to better explain it, and rather than using the number, say disposition of the "key concept" matches the majority of "supporting details"

```markdown
### Zooming in and out

Each `key concept` is a `detail` that focuses broadly on the position of the 
`party` in an important category. Narratively they zoom out and share more 
of the big picture. `Key concepts` are wide-reaching - they create an assumed 
default for anything related to that category that isn't already a `supporting 
detail`.

`Supporting details` add color to `key concepts` by zooming in on specific people, 
places, or things related to the party in some way (for example, a relationship with 
a specific `world force`).
```